python/tests/test_ecloutput.py

```diff
@@ -271,6 +271,137 @@ def test_write_lists(self):
             os.remove(testFile)
 
 
+    def test_write_strings_binary(self):
+
+        testFile = test_path("data/TMP1.INIT")
```

Comment:
These two test are 99% equal - it should be possible to refactor to reuse the body of the test and then just call it twise with formatted and unformatted output.

Reply:
For tests which write data to disk I consider it clean to make a temporary directory; something like
```python
import tempfile


with tempfile.TemporaryDirectory() as tmpdir:
       test_file = os.path.join(tmpdir, "TMP1.INIT")
       out1 = EclOutput(test_file)
       ....

#When the context has ended the temporary directory is removed.
```

...     print('created temporary directory', tmpdirname)

Reply:
I ended up using  nested function to avoid duplication of code. I also using tempfile as you show for all 7 tests in test_ecloutput.py.